pil_analyzer/src/pil_analyzer.rs

```diff
@@ -539,12 +539,11 @@ impl<'a, T: FieldElement> ExpressionProcessor<'a, T> {
             PExpression::Reference(poly) => {
                 if poly.namespace().is_none() && self.local_variables.contains_key(poly.name()) {
                     let id = self.local_variables[poly.name()];
-                    assert!(!poly.shift());
```

Comment:
I guess this information is not available here anymore?

Reply:
I moved this to the Next unary operator, it is not parsed in combination with references anymore. You can now also do things like `(x + f(t))'`.

Reply:
Yes, `'` is a linear operator.